### PR TITLE
[PFT-218] Integrate initial subgraph logic to the SDK

### PR DESCRIPTION
**Pull request by sudeepb02**

### CODE REVIEW

File: src/subgraph/orders/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// Get all order by a user address 
+export const getAllOrdersByUserAddress = async (chainId: Chain, userAddress: string, count: number = 10) => {
+    const subgraphEndpoint = getSubgraphEndpoint(chainId)
+
+    const subgraphResponse = await request(subgraphEndpoint, fetchOrdersByUserQuery(userAddress, count));
+    return mapOrdersArrayToInterface(subgraphResponse);
+}
+
+// Get all pending orders that are not yet settled/cancelled or expired 
+export const getAllPendingOrders = async (chainId: Chain, timestamp: number = Date.now() / 1000, count: number = 10) => {
```
Comment (wafflemakr) [inline src/subgraph/orders/index.ts]: Default value for timestamp is wrong. Need to add Math.floor()

File: src/subgraph/positions/subgraphQueries.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { gql } from "graphql-request";
+
+export const fetchPositionsByUserQuery = (userAddress: string, count: number = 10) =>
```
Comment (wafflemakr) [inline src/subgraph/positions/subgraphQueries.ts:3]: `count` param not used. I guess it should be in `first:${count}`

**Review by wafflemakr — CHANGES_REQUESTED** [review]
Minor changes

File: src/subgraph/orders/index.ts (hunk 1 above)
Comment (sudeepb02) [reply src/subgraph/orders/index.ts]: Fixed

File: src/subgraph/positions/subgraphQueries.ts (hunk 2 above)
Comment (sudeepb02) [reply src/subgraph/positions/subgraphQueries.ts:3]: Fixed

**Review by wafflemakr — APPROVED** [review]
LGTM

**Pull request merged**